code/index - Copy.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Camden Road Studios</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+  </head>
+  <body>
+    <!-- video or image as a header is cool :) -->
+    <!--<nav>
+      <a href="#">home</a>
+      <a href="#">about</a>
+      <a href="#">blog</a>
+      <a href="#">contact</a>
+    </nav> -->
+
+    <header class="header">
+      <h1 class="header-text">Camden Road Studios</h1>
+      <h2>write - record - release</h2> 
+      <div class="video-container">
+        <video autoplay loop muted> <!-- poster="http://" -->
+            <source src="./videos/video1.mp4" type="video/mp4">
+        </video>
+      </div>
+    </header>
+
+    <!-- Signup form -->
+    
+    <section class="main-container">
+      <h3>Sign up and book a session</h3>
+        <div class="flex-box">
+      <form 
+          class="form" 
+          action="http://httpbin.org/anything" 
+          method="POST"
+      >
+        <div class="form-group">
+          <label for="fullName">Name</label>
+          <input 
+              type="text" 
+              id="fullName" 
+              placeholder="Full name" 
+              name="fullNameValue" 
+              required 
+          />
```

Comment:
A for structure again - keep it up, and also great that you've used label correct together with the inputs!